main.css

```diff
@@ -1,138 +1,137 @@
-    font-size: 16px;
-    color: #344563;
-    line-height: 24px;
-  }
-  
-  .img1 {
-    position: absolute;
-    top: 5px;
-  }
-  
-  .img2 {
-    position: absolute;
-    top: 5px;
-    left: 127px;
-  }
-  
-  .img3 {
-    position: absolute;
-    top: 40px;
-    left: 340px;
-  }
-  
-  .img4 {
-    position: absolute;
-    top: 15px;
-    left: 129px;
-  }
-  
-  .img5 {
-    position: absolute;
-    top: -15px;
-    left: 320px;
-  }
-  
-  .img8 {
-    position: absolute;
-    top: 0 px;
-    left: 135px;
-  }
-  
-  .img7 {
-    position: absolute;
-    top: 0 px;
-    left: 185px;
-  }
-  
-  .text-bottom2 li {
-    list-style: none;
-  }
-  
-  .rectangle-item h2 {
-    font-size: 16px;
-    font-weight: 500;
-    color: #7f8cff;
-    line-height: 24px;
-  }
-  
-  .text-bottom2 {
-    display: flex;
-    left: 65px;
-  }
-  
-  .text-bottom2 ul {
-    left: -60px;
-  }
-  
-  .social {
-    padding: 12px;
-    margin-left: -12px;
-    margin-top: -9px;
-  }
-  
-  .rectangle-item {
-    display: flex;
-  }
-  
-  .rectangle-item img {
-    height: 15px;
-    float: right;
-    margin-top: 32px;
-    margin-left: 189px;
-  }
-  
-  .dimg {
-    height: 35px;
-    width: 40px;
-    margin-left: 294px;
-  }
-  
\ No newline at end of file
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  background-color: #f3f3f3;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow: hidden;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100vw;
+  margin-top: 50px;
+}
+
+.active {
+  margin-left: 30px;
+}
+
+nav {
+  top: 50px;
+  left: 24px;
+}
+
+.topnav {
+  display: flex;
+  justify-content: space-between;
+  width: 90%;
+}
+
+.head-text {
+  width: 375px;
+  top: 114px;
+  position: absolute;
+  left: 24px;
+}
+
+.head-text h1 {
+  font-size: 40px;
+  font-weight: 700;
+  color: #172b4d;
+  line-height: 52px;
+}
+
+.head-text p {
+  font-size: 16px;
+  color: #344563;
+  line-height: 24px;
+}
+
+.img1 {
+  position: absolute;
+  top: 5px;
+}
+
+.img2 {
+  position: absolute;
+  top: 5px;
+  left: 127px;
+}
+
+.img3 {
+  position: absolute;
+  top: 40px;
+  left: 340px;
+}
+
+.img4 {
+  position: absolute;
+  top: 15px;
+  left: 129px;
+}
+
+.img5 {
+  position: absolute;
+  top: -15px;
+  left: 320px;
+}
+
+.img8 {
+  position: absolute;
+  top: 0 px;
+  left: 135px;
+}
+
+.img7 {
+  position: absolute;
+  top: 0 px;
+  left: 185px;
+}
+
+.text-bottom2 li {
+  list-style: none;
+}
+
+.rectangle-item h2 {
+  font-size: 16px;
+  font-weight: 500;
+  color: #7f8cff;
+  line-height: 24px;
+}
+
+.text-bottom2 {
+  display: flex;
+  left: 65px;
+}
+
+.text-bottom2 ul {
+  left: -60px;
+}
+
+.social {
+  padding: 12px;
+  margin-left: -12px;
+  margin-top: -9px;
+}
+
+.rectangle-item {
+  display: flex;
+}
+
+.rectangle-item img {
+  height: 15px;
+  float: right;
+  margin-top: 32px;
+  margin-left: 189px;
+}
+
+.dimg {
+  height: 35px;
+  width: 40px;
+  margin-left: 294px;
+}
```

Comment:
**Nice work, great trial** :+1: 
- However, there are a few things that I believe you can change in order to have a more aesthetic and more accessible portfolio page in order to impress and show your excellent skills.

_Here is what a comparison between your work and the requirement looks like:_

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/234646615-c9de58e4-dacb-42eb-987f-05d6635b87b7.png)|![image](https://user-images.githubusercontent.com/99715640/234646721-e3ec49c7-3056-46a9-bcb6-30766ecf2f4c.png)|

- Let's see how we can improve this design by breaking it up into several parts/components since we don't have access to annotate your `index.html` file (which would have been easier):
  - _In the Header section_:
    - [ ] There seems to be too much space between the `top` and the content. Kindly reduce the `margin` or `padding` that's causing too much space. In case you left the space for mobile icons or a notification bar, there's no need for this since each user's device will have this feature. Let's ignore the mobile in-built notifications and just start with the header.
    - [ ] The hamburger icon is not showing because of a broken link. Kindly check your filename and fix this bug.
    - [ ] [OPTIONAL] It is proper semantics and an addition to sites's accessibility when `img` tags have the `alt` attributes which tells screen readers what image is all about, or for users with poor internet due to any cause.
  - In the `main` content - section:
    - [ ] Kindly observe how the "Glad to see you." is on a separate line in the project requirement. This makes the reading more organized, and understandable and give a good user experience. This can be implemented by using `<br>` (line break tag) in the `index.html` or with the use of `width` properties in css.
    - [ ] All the images in this section except for the social icons are purely for good designs and aesthetics. They have no specific meaning or enhance the content in any way, and when the case is this, those images are better introduced as `background-image`s. 
   
![image](https://user-images.githubusercontent.com/99715640/234658449-d07189b5-df90-4647-9379-9c3d3961b133.png)
  - Here's a hint: _If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info._
    - [ ] Kindly exercise care when importing the images so that you won't have any broken images on your page, which will have a huge negative impact on the design and the user experience.
  - At the bottom where we have the social icons:
    - [ ] As you can see, we have too much space than the requirement. Kindly stick to the design as best as possible. Since you have a wireframe that guides you and gives you values for your CSS properties, you don't really have to improvise much. The space between "Let's connect" and the list of icons is too much for a good design.
    - [ ] Also, I suggest that you use a `align-items: baseline;` on your `.text-bottom2` so that all the icons can have a common (well aligned to the base) base.
    - [ ] Just as you can tell from my markings on your design, there is much too extraneous space surrounding the social icons. Kindly fit them with the project requirement as seen in Figma.
    - [ ] In the entire section from Figma's, I think you are missing a `border-bottom-left-radius`. How about adding that so that you can match the wireframe?